### ISSUE TITLE
Suggestion - possibility to inherit theme

### ISSUE THREAD

**Issue by netdown** [marmelab/react-admin#3895]
**Is your feature request related to a problem? Please describe.**
If I am not mistaken, currently - when integrating ra into an app - there is no way to inherit a MUI theme from a parent ThemeProvider.

**Describe the solution you'd like**
I suppose somewhere a ThemeProvider is created. This could be disabled by setting theme to false.

**Describe alternatives you've considered**
I must nest the same theme by setting the theme param on the Admin component.

**Comment by fzaninotto** [comment]
Let me rephrase your problem: you are integrating ra into another app that already has a custom theme passed to a custom ThemeProvider. You don't want to pass that custom theme to react-admin, too. 

Why?

**Comment by netdown** [comment]
I think it is just a waste of source code lines and resources. And if I am not mistaken, the situation must be the same without a custom theme - the default theme is injected two times. You would not use 3 dom elements for some if 2 was enough, would you?

**Comment by fzaninotto** [comment]
OK, I understand, thanks. React-admin won't support it, though. You'll have to pass the theme to `<Admin>` even though you already initialized a `<ThemeProvider>.`
